src/devhub/index.html

```diff
@@ -49,15 +54,15 @@ <h3>Links</h3>
         <h3>Main branch fuzzes</h3>
         <table>
           <tr>
-            <td class="ok" style="padding-right: 16px;">ok</td>
+            <td class="ok" style="padding-right: 16px;" title="This many fuzzes didn't have any failure.">ok</td>
             <td><span id="fuzz-ok">N/A</span></td>
           </tr>
           <tr>
-            <td class="fail" style="padding-right: 16px;">fail</td>
+            <td class="fail" style="padding-right: 16px;" title="This many fuzzes had a failure.">fail</td>
```

Comment:
Perhaps, 

"The number of failed fuzz seeds on any branch in the last N days.”

Reply:
(Or I might be misunderstanding what this is!)